apps/web/src/features/safe-shield/components/SafeShieldContent/index.tsx

```diff
@@ -53,26 +51,14 @@ export const SafeShieldContent = ({
       <Box
         sx={{ border: '1px solid', borderColor: 'background.main', borderTop: 'none', borderRadius: '0px 0px 6px 6px' }}
       >
-        {loading ? (
-          <SafeShieldAnalysisLoading />
-        ) : error ? (
-          <SafeShieldAnalysisError error={error} />
-        ) : allEmpty ? (
-          <SafeShieldAnalysisEmpty />
-        ) : null}
+        {loading ? <SafeShieldAnalysisLoading /> : allEmpty ? <SafeShieldAnalysisEmpty /> : null}
 
         <Box display={loading || error ? 'none' : 'block'}>
```

Comment:
I think we also need to remove error from here, otherwise the results (and component-specific error results) would not be rendered in case there is any error. Or am I missing something here?
```suggestion
        <Box display={loading ? 'none' : 'block'}>
```

Reply:
yes for sure we don't need it! More one leftover from last rebase 🥹 Thanks for catching